src/components/sidebar/custom/CustomDialog.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, Fragment } from 'react';
+import { Button } from '@material-ui/core';
+import TextField from '@material-ui/core/TextField';
+import { useDispatch } from 'react-redux';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Checkbox from '@material-ui/core/Checkbox';
+import { makeStyles, ThemeProvider } from '@material-ui/styles';
+
+const useStyles = makeStyles({
+  classDescription: {
+    display: 'flex',
+    margin: 15,
+  },
+  schedule: {
+    margin: 20,
+  },
+  button: {
+    display: 'flex',
+    alignContent: 'space-between',
+    margin: 15,
+  },
+  time: {
+    justifyContent: 'space-between',
+    margin: 15,
+  },
+});
+
+
+export default function CustomDialog() {
+  const classes = useStyles();
+  const [state, setState] = React.useState({
+    monday: false,
+    tuesday: false,
+    wednesday: false,
+    thursday: false,
+    friday: false,
+  });
+
+  const handleChange = name => (event) => {
+    setState({ ...state, [name]: event.target.checked });
+  };
+
+  const { monday, tuesday, wednesday, thursday, friday } = state;
+  const error = [monday, tuesday, wednesday, thursday, friday].filter(v => v).length < 1;
+
+  return (
+    <Fragment>
+      <form noValidate autoComplete="off">
```

Comment:
I think I was unclear in the ticket description - we won't actually need a `form` element for this. What we can do is construct a state (similar to how you have the day of the week right now), and upon clicking of the submit button later, dispatch an action that alters the Redux store state. We can talk more on Slack if this is confusing.

Reply:
Actually, scratch what I said - a form is good, but we don't need the vanilla HTML `form` element, we can stick everything under the `FormControl` like you have below. is there a reason why these two first `TextFields` are outside of the `FormControl`?